src/js/scriptlets/dom-inspector.js

```diff
@@ -264,15 +264,17 @@ const domLayout = (function() {
                 continue;
             }
             // sibling
-            if ( node.nextElementSibling === null ) {
-                do {
-                    node = stack.pop();
+            if (node instanceof Element) {
```

Comment:
@gorhill just realized that this should probably be `if (node !== null && node.nodeType === 1)` for better perf and cross-realm elements compatibility if they could somehow get here. will make a pr in a moment

Reply:
submitted a pr to fix this: https://github.com/gorhill/uBlock/pull/3942